### PR TITLE
Update EBS CSI driver KEP for Beta release

### PR DESCRIPTION
**Pull request by leakingtapan**
xref: #630

### CODE REVIEW

File: keps/sig-aws/20181127-aws-ebs-csi-driver.md [hunk 1]
```diff
@@ -64,15 +68,52 @@ Operator creates a storage class that defines EBS CSI driver as provisioner. Dev
 #### Volume Scheduling
 Operation creates StorageClass with  volumeBindingMode = WaitForFirstConsumer. When developer deploys a Pod that has PVC that is trying to claim for a PV, a new PV will be created, attached, formatted and mounted inside Pod&#39;s container by the EBS CSI driver. Topology information provided by EBS CSI driver will be used during Pod scheduling to guarantee that both Pod and volume are collocated in the same availability zone.
```
[the remark this answers is not in the document]
Comment (leakingtapan) [reply keps/sig-aws/20181127-aws-ebs-csi-driver.md]: Cannot just type `'` in markdown, `&#39;` is the apostrophe

File: keps/sig-aws/20181127-aws-ebs-csi-driver.md [hunk 2]
```diff
@@ -64,15 +68,52 @@ Operator creates a storage class that defines EBS CSI driver as provisioner. Dev
...
+This assumes user is already using some version of the driver.
+
+### Upgrade
+This assumes user is already using Kubernetes 1.13 cluster. Otherwise, the existing cluster needs to be upgraded to 1.13+ in order to install the driver. Formal cluster upgrade process should be followed for upgrading cluster.
+
+Driver upgrade should be performed one version at a time. This means, if the current driver verison is 0.1, it can be upgraded to version 0.2 by following the upgrade process. And if the driver version that is required to upgrade to is 0.3, it should be upgraded to 0.2 first.
+
+To upgrade the driver, perform following steps:
+1. Delete the old driver controller service and node service along with other resources including cluster roles, cluster role bindings and service accounts.
+1. Deploy the new driver controller service and node service along with other resources inclluding cluster roles, cluster role bindings and service accounts.
```
[the remark this answers is not in the document]
Comment (leakingtapan) [reply keps/sig-aws/20181127-aws-ebs-csi-driver.md]: numeric ordering will be numbered by markdown automatically

**Comment by d-nishi** [comment]
@leakingtapan thanks for resolving all the comments.

/lgtm
/approve

**Pull request merged**